File: Java/Palindrome/PalindromeChecker.java
import java.util.Scanner;
/**
 * @author brian
 *
 * Checks if the user's input [a word/phrase] is a palindrome
 * - a word/phrase that remain the same when read in reverse
 */
public class PalindromeChecker extends FindPalindrome
{
	public static void main(String[] args) 
	{
		Scanner input = new Scanner(System.in);
		System.out.print("Check if a word/phrase is a palindrome: ");
		String text = input.nextLine();
		
		boolean palin = palindrome(text);
		System.out.println(palin);
		
		if(palin)
		{
			System.out.println(text + " is a palindrome!");
		}
		else
		{
			System.out.println(text + " is not a palindrome :(");
		}
		
		input.close();
	}
}

File: Python/PalindromeChecker.py
'''
Created on May 29, 2020

@author: brian
'''

"""
Python program that checks if a word
is a palindrome - a word/phrase that
will remain the same when read in 
reverse.
"""
def is_palindrome(word):
    palin = ''.join(reversed(word))
    
    if(word == palin):
        return True
    return False

inputText = raw_input("Enter a word: ");

palindrome = is_palindrome(inputText)

print(palindrome)

if(palindrome):
    print(inputText + ' is a palindrome :)')
else:
    print(inputText + ' is not a palindrome :(')